### PR TITLE
Add a Powershell Script for Container Preview of Website on Windows

### PR DESCRIPTION
**Pull request by Gauravpadam**
closes https://github.com/kubernetes/website/issues/42861

This adds a Powershell script for Windows Users
They can now deploy containers without explicitly invoking bash/wsl

### CODE REVIEW

File: hugo.toml [hunk 1]
```diff
@@ -8,6 +8,7 @@ theme = ["docsy"]
 contentDir = "content/en"
 defaultContentLanguage = "en"
 defaultContentLanguageInSubdir = false
+i18nDir = "./data/i18n"
```
Comment (Gauravpadam) [inline hugo.toml:11]: This is moved under language settings as the script was not able to catch the directory, (All build settings remain fully functional)

**Comment by Gauravpadam** [comment]
Good for review now

**Comment by marosset** [comment]
Are there any instructions for running this?

I tried to run this on a windows machine (where I already have WSL , and docker running in that wsl instance set up) and I'm getting this error early on

> PS C:\src\github.com\marosset\website> .\windowsmake.ps1
/bin/bash: scripts/hash-files.sh: /bin/sh^M: bad interpreter: No such file or directory

**Comment by Gauravpadam** [comment]
I might have to close this, Accidently force pushed an incorrect rebase.

**Pull request closed without merging**